services/ad-tech/src/routes/common/report-router.ts

```diff
@@ -30,6 +37,29 @@ import {getTemplateVariables} from '../../lib/template-utils.js';
  */
 export const ReportRouter = express.Router();
 
+/** Sets ARA source registration headers if request is eligible. */
+const setAttributionReportingHeadersIfEligible = (
```

Comment:
Can this just be `setAttributionReportingHeaders`? Then can we add an else statement to catch if the eligibility header is not included in the request? So the "if eligible" piece is inferred and caught if the header isn't provided

Reply:
Ack, done.